tls_for_test.go

```diff
@@ -0,0 +1,185 @@
+package main_test
+
+/*
+ * This file implements a certificate authority and certs for testing otel-cli's
+ * TLS settings.
+ *
+ * Do NOT copy this code for production systems. It makes a few compromises to
+ * optimize for testing and ephemeral certs that are totally inappropriate for
+ * use in settings where security matters.
```

Comment:
is there any way to make this more forceful than a comment?
off the top of my head, perhaps some kind of output from the module that would flag the issue to an end user?
or maybe also add code comments that point out the specific compromises a cargo-cult-er needs to change?
and the last thought was adding some kind of friction here to make it _hard_ to misuse, though I have no concrete ideas on how to go about that.

Reply:
oh and one last thought was to add something akin to a comment in the output files so that anyone viewing their contents would investigate and find the comments in code